src/libraries/System.Private.CoreLib/src/Microsoft/Win32/SafeHandles/SafeFileHandle.Unix.cs

```diff
@@ -33,6 +33,13 @@ private SafeFileHandle(bool ownsHandle)
 
         internal bool CanSeek => !IsClosed && GetCanSeek();
 
+        private bool? _supportsRandomAccess;
```

Comment:
`_canSeek` is a `volatile NullableBool` because `bool?` is not thread safe.
This should be the of the same type.

Reply:
Right, thanks.